packages/plans-grid-next/src/components/features.tsx

```diff
@@ -96,8 +96,16 @@ const PlanFeatures2023GridFeatures: React.FC< {
 					  ( currentFeature.getSlug() === FEATURE_CUSTOM_DOMAIN && paidDomainName ) ||
 					  ! currentFeature.availableForCurrentPlan;
 
+				const featureSlug = currentFeature.getSlug();
+				const featuresWithMinHeight = [
+					'support-from-experts',
+					'priority-24-7-support',
+					'upload-video',
+				];
+
 				const divClasses = clsx( '', getPlanClass( planSlug ), {
 					'is-last-feature': featureIndex + 1 === features.length,
+					'has-min-height': featuresWithMinHeight.includes( featureSlug ),
```

Comment:
Temporary fix: The feature grouping mechanism would normally address this, but defining the groups while keeping the design's feature order is proving complicated. We can revisit once we have settled on the final features.